packages/renderer/src/lib/images/ServiceIcon.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+export let size = '1em';
+export let solid = false;
+
+let boxStyle = 'fill: currentColor;';
+if (!solid) {
+  boxStyle = 'stroke-miterlimit: 10; stroke-width: 1; stroke: currentColor; stroke-opacity: 1;';
```

Comment:
can't we use tailwind classes ?  `stroke-1` `stroke-current`

Reply:
Updates:
- I realized stroke-width and stroke-opacity were just set to the defaults, so I removed those (and I bet we could in several other svgs too).
- Changed "fill: none" to fill-none class.
- Realized miterlimit wasn't having any visible effect even at huge sizes, so removed (again, could probably do in other icons too).
- Using fill-current and stroke-current don't work (goes to black), so I left those as styles.

I suspect the 'current' styles need to be defined in our palette or something, so IMHO this is as far as we can go for now.